news/2022-07-28-beatmap-spotlights-spring-2022-conclusion.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Beatmap Spotlights Spring 2022 Conclusion & Players' Pick Voting"
+date: 2022-07-28 16:00:00 +0000
+---
+
+The Spring 2022 Season of the Beatmap Spotlights has come to an end⁠ — come check out the top 50 for each game mode, and vote on your favourite beatmaps from the season! Join us on Summer 2022 Season launch livestream as well.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Continuing the trend from past seasons, Spring 2022 playlists gathered nearly **2900** players either competing for fun or jumping on a grindwagon to gain this *Rhythm Incarnate* rank. Since the last season's Players' Pick turned out gripping for both players and mappers, we will be running it again this season, read on below for that and even more exciting new stuff!
```

Comment:
```suggestion
Continuing the trend from past seasons, Spring 2022 playlists gathered nearly **2900** players either competing for fun or jumping on a grindwagon to gain this *Rhythm Incarnate* rank. Since the last season's Players' Pick turned out gripping for both players and mappers, we will be running it again this season — read on below for that and even more exciting new stuff!
```